keyboards/mechwild/sugarglider/sugarglider.c

```diff
@@ -0,0 +1,441 @@
+        } else {
+            oled_write(get_u16_str(g_tapping_term, ' '), false);
+        }
+#endif
+        clear_screen = true;
+    } else {
+        if (clear_screen == true) {
+            oled_clear();
+            oled_render();
+            clear_screen = false;
+        }
+        render_logo();
+        oled_set_cursor(8,1);
+        switch (get_highest_layer(layer_state)) {
+            case 0:
+                oled_write_P(PSTR("Base Layer"), false);
+                break;
+            case 1:
+                oled_write_P(PSTR("Lower"), false);
+                break;
+            case 2:
+                oled_write_P(PSTR("Raise"), false);
+                break;
+            case 3:
+                oled_write_P(PSTR("Adjust"), false);
+                break;
+            case 4:
+                oled_write_P(PSTR("Mouse Layer"), false);
+                break;
+#ifdef STENO_ENABLE
+            case 5:
+                oled_write_P(PSTR("Steno Layer"), false);
+                break;
+#endif
+            default:
+                oled_write_P(PSTR("Layer ?"), false);    // Should never display, here as a catchall
+        }
+        led_t led_state = host_keyboard_led_state();
+        oled_set_cursor(8,0);
+        oled_write_P(led_state.num_lock ? PSTR("NUM ") : PSTR("    "), false);
+        oled_write_P(led_state.caps_lock ? PSTR("CAP ") : PSTR("    "), false);
+        oled_write_P(led_state.scroll_lock ? PSTR("SCR") : PSTR("    "), false);
+#ifdef POINTING_DEVICE_ENABLE
+        oled_set_cursor(8,2);
+        oled_write_P(PSTR("DPI:"), false);
+        oled_write(get_u16_str(dpi_array[keyboard_config.dpi_config], ' '), false);
+#endif
+#ifdef DYNAMIC_TAPPING_TERM_ENABLE              // only display tap info if it is being configured dynamically
+        oled_set_cursor(8,3);
+        oled_write_P(PSTR("TAP:"), false);
+        if (keyboard_config.dt_term_config < 0) {
+            oled_write_P(PSTR("Off  "), false);
+        } else {
+            oled_write(get_u16_str(g_tapping_term, ' '), false);
+        }
+#endif
+        clear_screen_art = true;
+    }
+    return true;
+}
+#elif defined(OLED_ENABLE) && ! defined(OLED_DISPLAY_128X64)
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    return OLED_ROTATION_270;       // flips the display 270 degrees
+}
+
+static void render_logo(void) {     // Render MechWild "MW" Logo
+    static const char PROGMEM logo_1[] = {0xCC, 0xCD, 0xCE, 0xCF, 0x00};
+    static const char PROGMEM logo_2[] = {0xD0, 0xD1, 0xD2, 0xD3, 0xD4, 0x00};
+    static const char PROGMEM logo_3[] = {0xD5, 0xD6, 0xD7, 0xD8, 0x00};
+    static const char PROGMEM logo_4[] = {0xDE, 0xD9, 0xDA, 0xDB, 0x00};
+    
+    oled_set_cursor(0,0);
+    oled_write_P(logo_1, false);
+    oled_set_cursor(0,1);
+    oled_write_P(logo_2, false);
+    oled_set_cursor(0,2);
+    oled_write_P(logo_3, false);
+    oled_set_cursor(0,3);
+    oled_write_P(logo_4, false);
```

Comment:
```suggestion
    oled_write_P(logo_1, false);
    oled_advance_page(true);
    oled_write_P(logo_2, false);
    oled_advance_page(true);
    oled_write_P(logo_3, false);
    oled_advance_page(true);
    oled_write_P(logo_4, false);
```

Reply:
These changes just don't show the lines as expected and it breaks the whole display when I try and write over other parts of the display, such as with the dynamic text, layers, etc. It works well if I don't try and write over anything.